utils_test.go

```diff
@@ -313,6 +322,28 @@ func TestRecordMetric(t *testing.T) {
 			recordErr:      driver.ErrSkip,
 			expectedStatus: "ok",
 		},
+		{
+			name: "metric with instrumentAttributesGetter",
+			args: args{
+				cfg:    newConfig(WithInstrumentAttributesGetter(dummyAttributesGetter)),
+				method: MethodConnQuery,
+				query:  "example query",
+			},
+			recordErr:      nil,
+			expectedStatus: "ok",
+			expectedAttr:   defaultattribute,
+		},
+		{
+			name: "metric with errorAttributesGetter",
```

Comment:
Could we add a test that uses `WithErrorAttributesGetter`, and the `recordErr` is nil?